src/lib/services-manager/dialogs/services_manager.rb

```diff
@@ -464,13 +464,13 @@ def start_mode_button_handler(mode)
       def show_details_button_handler
         self.finish = false
 
-        service = selected_service_name
+        service = selected_service
         full_info = ServicesManagerService.status(service)
-        x_size = full_info.lines.collect{|line| line.size}.sort.last
+        x_size = full_info.lines.collect(&:size).sort.last
```

Comment:
In order to be confident, please, replace `collect` with `map` (I know it was not modified by you, but give you are touching that line...) :)